javascript/javascript1/week1/dog-age-calci.js

```diff
@@ -0,0 +1,18 @@
+let dogYearOfBirth = 2010;
+let dogYearFuture = 2022;
+let dogYear = dogYearFuture - dogYearOfBirth;
```

Comment:
As long as the value of these variables is not changed, it is preferred to use `const`